### PR TITLE
[WIP] Clean up readme and migrate materials to IPFS docs

### PR DESCRIPTION
**Pull request by jessicaschilling**
1. Cleans up readme overall:
- Removes/replaces dead or archived links
- Reorganizes for clarity
- Tightens language for length

2. Moves relevant materials into docs (see https://github.com/ipfs/ipfs-docs-v2/pull/148 ):
- Implementation status page
- Roadmap page

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@hsanjuan -- can you please review? This should make some substantial progress in turning this repo into a more evergreen guidance document.

Note that while I removed or updated a number of dead or archived links, you'll likely still want to add links to some of the newer repos/projects that have emerged since this document was last looked at in detail.

**Comment by jessicaschilling** [comment]
@hsanjuan Material slated to migrate to docs (roadmap, implementation status) is now live in docs, so we can merge this whenever you're content with it.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,16 +0,0 @@
-For QUESTIONS, DISCUSSIONS and SUPPORT issues, go to:
```
Comment (hsanjuan) [inline ISSUE_TEMPLATE.md:1]: We should keep this? Unless we are de-activating the issues tab altogether...

File: README.md [hunk 2]
```diff
@@ -1,185 +1,167 @@
...
-[![Forum](https://img.shields.io/badge/Discourse-%23Discussion forums-blue.svg)](https://discuss.ipfs.io)
+[![Forum](https://img.shields.io/badge/Discourse-%23Discussion%20forums-blue.svg)](https://discuss.ipfs.io)
 [![Matrix](https://img.shields.io/badge/matrix-%23ipfs%3Amatrix.org-blue.svg?style=flat-square)](https://matrix.to/#/#ipfs:matrix.org)
 [![IRC](https://img.shields.io/badge/irc-%23ipfs-brightgreen.svg?style=flat-square)](https://webchat.freenode.net/?channels=ipfs)
 [![Discord](https://img.shields.io/discord/475789330380488707?color=blueviolet&label=discord&style=flat-square)](https://discord.gg/24fmuwR)
 [![Changelog #204](https://img.shields.io/badge/changelog-%23204-lightgrey.svg)](https://changelog.com/204)
 
-Welcome to IPFS! Why not [watch a video demo](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
-
-**Please post questions and ideas at [https://discuss.ipfs.io](https://discuss.ipfs.io) .**
```
Comment (hsanjuan) [inline README.md:13]: We should funnel users with questions or support needs to Discourse. We should have very prominent links on this page, at the top, on where to get help. If not like this, perhaps as first item of the TL;DR...

File: README.md [hunk 3]
```diff
@@ -1,185 +1,167 @@
...
-- bittorrent-based bandwidth distribution
+IPFS is a **distributed file system** that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single BitTorrent swarm exchanging Git objects. You can read more about its origins in the paper [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true).
 
-IPFS has a name service:
-- IPNS, an SFS inspired name system
-- global namespace based on PKI
-- serves to build trust chains
-- compatible with other NSes
-- can map DNS, .onion, .bit, etc to IPNS
+IPFS is becoming a **new major subsystem of the internet**. If built right, it could complement or replace HTTP. It could complement or replace even more. Let's go point-by-point into how.
```
Comment (hsanjuan) [inline README.md:51]: I'm going to scroll quickly through these and assume they are copy-paste from somewhere where this has been reviewed.

File: README.md [hunk 4]
```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
...
-
-There are plethora of IPFS Meetups all over the world now. Find yours at the [Community Repo](https://github.com/ipfs/community#events)
-
-We encourage and support IPFS Meetups, please let us know if you would like to start one. Feel free to organize yourself through [the Community discussions](https://github.com/ipfs/community/issues) and [to advertise events in the main repository](https://github.com/ipfs/ipfs/issues).
-
-#### Tools
+- [specs](https://github.com/ipfs/specs) - Specifications on the IPFS protocol.
+- [papers](papers) - Academic papers on IPFS.
+- [IPFS Docs: Further Reading](https://docs-beta.ipfs.io/concepts/further-reading/academic-papers) - Papers to read to understand IPFS and its underlying technologies.
+- [notes](https://github.com/ipfs/notes) - The IPFS collaborative notebook for research, used for discussing known open IPFS problems, nascent issues, proposals, or features.
```
Comment (hsanjuan) [inline README.md:225]: Maybe we should leave /notes out and name the section `Specs and Papers` only. We need to start phasing out /notes, or at least making it a more specific/focused discussion place.

File: README.md [hunk 5]
```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
...
 
-Installing
+### Installation and update tools
 
 - [install-go-ipfs](https://github.com/ipfs/go-ipfs#install) - Install go-ipfs shell script.
-- [install-js-ipfs](https://github.com/ipfs/js-ipfs#install) - Install js-ipfs through npm or a script tag.
+- [npm-go-ipfs](https://github.com/ipfs/npm-go-ipfs) - Install go-ipfs from NPM.
```
Comment (hsanjuan) [inline README.md:230]: This is not overly maintained I think, best to be placed at the bottom of this list.

File: README.md [hunk 6]
```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
...
+- [dir-index-html](https://github.com/ipfs/dir-index-html) - Directory listing HTML.
 - [dnslink-deploy](https://github.com/ipfs/dnslink-deploy) - Automatically set DNS records on Digital Ocean.
-- [file-browser](https://github.com/ipfs/file-browser) - Generic IPFS file browser UI
-- [fs-stress-test](https://github.com/ipfs/fs-stress-test) - Stress testing IPFS filesystem capabilities.
+- [fs-stress-test](https://github.com/ipfs/fs-stress-test) - Stress-testing IPFS filesystem capabilities.
 - [js-ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl) - Control IPFS daemons from JavaScript.
 - [ipfs-hubot](https://github.com/ipfs/ipfs-hubot) - Hubot for IPFS uses.
 - [ipfs-blob-store](https://github.com/ipfs/ipfs-blob-store) - A place to buy blobs.
-- [Find an ever growing list on Awesome IPFS](https://awesome.ipfs.io/tools/)
+
```
Comment (hsanjuan) [inline README.md:244]: I cannot answer for the good maintenance of all of these. Also they are drops in an ocean of "other tools" (maybe the first drops in that ocean back in the day). Might as well kill the section.

File: README.md [hunk 7]
```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
...
+### Community infrastructure
```
Comment (hsanjuan) [inline README.md:245]: `Additional resources` ?

File: README.md [hunk 8]
```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
...
+
+- [distributions](https://github.com/ipfs/distributions) - Source code for the IPFS distributions website, https://dist.ipfs.io.
+- [infra](https://github.com/ipfs/infra) - Tools for maintaining infrastructure for the IPFS community.
+- [testground](https://github.com/ipfs/testground) - Tools for testing distributed software at scale
+- [website](https://github.com/ipfs/website) - Source code for the IPFS website, http://ipfs.io.
```
Comment (hsanjuan) [inline README.md:251]: Would be nice to have a link to IPFS Cluster somewhere in the readme, perhaps this list is an ok place.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jessicaschilling) [reply ISSUE_TEMPLATE.md:1]: I don't see any reason NOT to de-activate the issues tab; this readme does give specific guidance in the "Help and documentation" section about raising issues in their appropriate repos.

Unfortunately I don't have adequate permissions to disable issue creation. @hsanjuan, can you please do this?

File: README.md (hunk 2 above)
Comment (jessicaschilling) [reply README.md:13]: Was already in the TL;DR, but moved it to the top and boldfaced it.

File: README.md (hunk 3 above)
Comment (jessicaschilling) [reply README.md]: All the material in this section was already in the old readme -- I just consolidated two places where we were essentially saying the same thing twice.

File: README.md (hunk 5 above)
Comment (jessicaschilling) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (jessicaschilling) [reply README.md]: Done.

File: README.md (hunk 6 above)
Comment (jessicaschilling) [reply README.md]: Concur. Done.

File: README.md (hunk 8 above)
Comment (jessicaschilling) [reply README.md:241]: Added link to Cluster - also a link to Shipyard in the sense that if we're now calling this "additional resources", that's another major additional resource.

File: README.md (hunk 7 above)
Comment (jessicaschilling) [reply README.md]: Done.

**Comment by jessicaschilling** [comment]
@hsanjuan - changes made, can you please have one final look? Also, I've not got enough rights to disable issue creation. Can you please do this?

File: README.md (hunk 2 above)
Comment (hsanjuan) [reply README.md:13]: wops

**Pull request merged**